yarn.lock

```diff
@@ -4,27 +4,27 @@
 
 "@ampproject/remapping@^2.2.0":
   version "2.2.1"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.2.1.tgz#99e8e11851128b8702cd57c33684f1d0f260b630"
+  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.1.tgz"
```

Comment:
I'm uncertain if it was an intended change - I think we should decide on using `npm` or `yarn`. I think we should keep with `yarn` to keep the current setup and consistency.

So, what I've noticed, is that `npm` would change `yarn.lock` to its coordinates if used. I'd suggest reverting changes from `yarn.lock` and use `yarn add @react-native-async-storage/async-storage --save`

Reply:
Thanks for this @wzieba. My bad, I did `npm install ...` and didn't think much about it. I'll use `yarn` from now on.